lib/data/replacements.json

```diff
@@ -226,6 +227,18 @@
       }
     ]
   },
+  "eslint-plugin-es-to-maintained-fork": {
+    "description": "Replace `eslint-plugin-es` with a maintained fork: `eslint-plugin-es-x`.",
+    "packageRules": [
+      {
+        "matchDatasources": ["npm"],
+        "matchPackageNames": ["eslint-plugin-es"],
+        "matchCurrentVersion": "^4.1.0",
+        "replacementName": "eslint-plugin-es-x",
+        "replacementVersion": "4.1.0"
```

Comment:
I've just noticed that version 4.1.0 doesn't exist in the fork. Should I use the older version 5.0.0 or the newer one?